docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -20,18 +20,19 @@ Zowe Version 2.11.0 contains the enhancements that are described in the followin
 
 #### Zowe CLI (Core)
 
+- Bumped Secrets SDK to `7.18.3`. It uses more reliable resolution logic for `prebuilds` folder; adds static CRT for Windows builds.
+- Updated daemon on MacOS to use universal binary which adds support for Apple Silicon.
```

Comment:
@adam-wolfe Is there an issue/PR we ancan add as a link here?

Reply:
https://github.com/zowe/zowe-cli/pull/1766